rskj-core/src/main/java/co/rsk/core/bc/FamilyUtils.java

```diff
@@ -156,11 +156,6 @@
             List<Block> uncles = store.getChainBlocksByNumber(ancestor.getNumber());
 
             for (Block uncle : uncles) {
-                // TODO quick fix, the block storage should be reviewed
-                if (uncle == null) {
-                    continue;
```

Comment:
are we sure we want to remove this?

Reply:
It was tackled in an upper layer at

https://github.com/rsksmart/rskj/pull/664/commits/de1d59bb7d2621dd985abf979537a8b23ae65565#diff-bc750f44b5f1043fbeffc83b42d24894R513